.travis.rosinstall

```diff
@@ -1,3 +1,6 @@
 - git:
     uri: https://github.com/ros/std_msgs.git
     local-name: ros/std_msgs
+- git:
+    uri: https://github.com/ros-industrial/industrial_ci.git
```

Comment:
> This job is using the file from the master branch.

Ah, you're absolutely right. We can't test unless you change the URL of the file.
Ok, we can see how this goes after merge.